### ISSUE TITLE
these 3 blocks

### ISSUE THREAD

**Issue by gego51** [jmoenig/Snap#2710]
why not group these 3 blocks into one because they are exactly the same 
![regroupper](https://user-images.githubusercontent.com/5407223/98235412-0b31c800-1f62-11eb-9736-64c1f2f0f9c1.PNG)

**Comment by jmoenig** [comment]
you're right, all rings are *semantically* identical, but their different inner shapes offer a *syntactical* hint which kind of block is expected, especially when we use them as input slots in customn blocks.

**Comment by gego51** [comment]
![Capture](https://user-images.githubusercontent.com/5407223/98250342-9a95a600-1f77-11eb-982a-15280732b811.PNG)
you can simplifie the custom block's editor, i have made a picture

**Comment by jmoenig** [comment]
Sure, but then we could also get by with only a single type of input slot. The point of these different shapes is to give hints to users what we expect them to place inside. This is about users, not about the evaluator.

**Comment by gego51** [comment]
ok
